modules/pubsub/src/main/resources/application.conf

```diff
@@ -35,6 +35,11 @@
     "sink": 3
   }
 
+  "monitoring": {
+    "metrics": {
+      "cloudwatch": false
+    }
+
```

Comment:
Sounds weird to have Cloudwatch settings for `enrich-pubsbub` but agreed that removing `Option` and `flatMap` makes the code nicer to read.

Reply:
Nicer to read, but more importantly less error prone.  [This line](https://github.com/snowplow/enrich/blob/3.0.1/modules/kinesis/src/main/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Sink.scala#L84) had a bug!  The last `true` should have been `false`.